docs/advanced-setup.md

```diff
@@ -31,3 +31,40 @@ as the watchdog.
 
 !!! Note
     The sd_notify communication between the bot and the systemd service manager will not work if the bot runs in a Docker container.
+
+## Logging to syslog or journald
+
+On many systems the bot can be configured to send its log messages to `syslog` or `journald`. The special values for the `--logfilename` option can be used for this:
+
+* `--logfilename journald` -- send log messages to `journald`. This needs the `systemd` python package installed as the dependency. Not available on Windows.
```

Comment:
"... dependency, which is not available on Windows."

Reply:
I meant the functionality itself (logging to journald) is not available on Windows, but the dependency as well...